utilities/fill_counties_api.py

```diff
@@ -0,0 +1,119 @@
+import csv
+import os
+import time
+import warnings
+from multiprocessing.pool import Pool
+from os.path import abspath, dirname
+from os.path import join as join_path
+
+import geojson
+import psycopg2
+import requests
+from dotenv import load_dotenv
+from initials import initials
+
+load_dotenv()
+
+db_config = {
+    "database": os.getenv("PSQL_DB"),
+    "user": os.getenv("PSQL_USER"),
+    "password": os.getenv("PSQL_PASSWORD"),
+    "host": os.getenv("PSQL_HOST"),
+    "port": os.getenv("PSQL_PORT"),
+}
+
+
+# Get geojson from geocode
+def get_api(geocodigo):
+    return f"https://servicodados.ibge.gov.br/api/v3/malhas/municipios/{geocodigo}?formato=application/vnd.geo+json"
+
+
+# Return data from API
+def read_json(geocodigo):
+    url = get_api(geocodigo)
+    status = 0
+    wait = 3
+    while status != 200 and wait <= 16:
+        resp = requests.get(url)
+        status = resp.status_code
+        time.sleep(wait)
+        wait *= 3
+    resp_data = resp.json()
+    return resp_data
+
+
+# Parse data
+# Compare cities(Nome_Município) in the csv file
+def county_polygon(county_code):
+    for feature in read_json(county_code)['features']:
+        if feature["properties"].get("codarea") == county_code:
+            return geojson.dumps(feature['geometry'])
+    warnings.warn("is not in this geojson: {}.".format(county_code))
+    return
```

Comment:
probably here it should return an empty string.

another point, is `return geojson.dumps(feature['geometry'])` correct?
shouldn't it be just  `return geojson.dumps(feature)` ?